provisioning_templates/PXEGrub2/pxegrub2_global_default.erb

```diff
@@ -7,12 +7,15 @@ name: PXEGrub2 global default
 default=0
 timeout=20
 
+# On Debian/Ubuntu grub2 does not have patch for loading MAC-based configs. Also due to bug
+# in RHEL 7.4 files are loaded with an extra ":" character at the end. This workarounds both
+# cases, make sure "regexp.mod" file is present on the TFTP. For more info see:
+# https://bugzilla.redhat.com/show_bug.cgi?id=1370642#c70
+insmod regexp
 regexp --set=1:m1 --set=2:m2 --set=3:m3 --set=4:m4 --set=5:m5 --set=6:m6 '^([[:xdigit:]]{1,2})\:([[:xdigit:]]{1,2})\:([[:xdigit:]]{1,2})\:([[:xdigit:]]{1,2})\:([[:xdigit:]]{1,2})\:([[:xdigit:]]{1,2})' "$net_default_mac"
 mac=${m1}-${m2}-${m3}-${m4}-${m5}-${m6}
 configfile=/grub2/grub.cfg-01-$mac
-if [ -f "$configfile" ]; then
-  source "$configfile"
-fi
+source "$configfile"
```

Comment:
@dLobatog watch out when backporting - the `if [ -f "$configfile" ]; then` was added yesterday but I found it was incorrect. So just ignore that, the only relevant bit here is "insmod regexp"

Reply:
@lzap We don't do backporting with community templates - they are synced on every minor release